### ISSUE TITLE
Help Wanted: Fix Console Log

### ISSUE THREAD

**Issue by GaryCraft** [GaryCraft/ptero-plutonium#10]
The latest version of the images and egg for Plutonium-T6 have a broken log, this does not affect the server's functionality in a significant way, but it does make it harder to keep track of the server status.

**Comment by ZEROPOINTBRUH** [comment]
Are we talking with iw4m or plutonium itself trying to write a log file onto the server?

**Comment by GaryCraft** [comment]
Plutonium Console Log

**Comment by GaryCraft** [comment]
GaryCraft_#4984

**Comment by ZEROPOINTBRUH** [comment]
@GaryCraft This error is nonrepairable as plutonium calls for a color code to print on the console.
![Color Codes](https://user-images.githubusercontent.com/50062288/159034732-c66a7de3-492a-4c49-94fe-180be49f1e51.png)

To fix this issue you would have to request plutonium to change their proprietary code.

**Comment by brolpzt** [comment]
But if you run the server directly on linux with wine, without docker, the colors work normally.

I tested it here and it only happens when the server runs in a container. Hence in pterodactyl as well.

The same for the order of the lines. It works 100% if you run it without being in a container.

I made a way here to print the console.log file in the dashboard console, but it's not the solution since it doesn't show ALL the information.
